regex-assembly/942180.ra

```diff
@@ -3,27 +3,47 @@
 
 ##!+ i
 
-\d[\"'`]\s+[\"'`]\s+\d
-^admin\s*?[\"'`]
-(?:/\*)+[\"'`]+\s?
-(?:/\*)+[\"'`]+\s?--
-(?:/\*)+[\"'`]+\s?#
-(?:/\*)+[\"'`]+\s?/\*
-(?:/\*)+[\"'`]+\s?{
-[\"'`]\s*?or[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?xor[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?div[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?like[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?between[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?and[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?[^\w\s]?=\s*?[\"'`]
-[\"'`]\W*?[+=]+\W*?[\"'`]
-[\"'`]\s*?[!=|][\d\s!=+-]+.*?[\"'`(].*?$
-[\"'`]\s*?[!=|][\d\s!=]+.*?\d+$
-[\"'`]\s*?like\W+[\w\"'`(]
+
+##!> define open-comment (?:/\*)
+
+##!> define quotes [\"'`]
+##!> define quotes-plus-digits [\d\"'`]
+##!> define comment-suffix (?:--|#|\/*|\{)
+##!    PB: Should comment-suffix also include "\*/" ?
+
+##!> define logical-operators or|xor|and
```

Comment:
Instead of using definitions for these, use one assemble block for each and put alternations on individual lines. Example:
```
##!> assemble
  div
  ##!=< math-operators
##!<
```
Then use it like this
```
...
##!> assemble
  ##!=> math-operators
##!<
```

It's a bit more verbose it places values on individual lines, which makes it much simpler to edit the lists.

I think, using a definition for `math-symbols` is fine though.